lib/commands/SpringBootBomRelease.ts

```diff
@@ -27,7 +27,7 @@ export class SpringBootBomRelease implements HandleCommand {
   @Parameter({
     displayName: "bom_version",
     // tslint:disable-next-line: max-line-length
-    description: "The upstream snowdrop release of the BOM for which the update is going to run (something like: 1.5.15.Final)",
```

Comment:
I would rather say something like: `description: "The full version string including qualifier (e.g. 2.1.10.Beta1, 1.5.19.SP3, 2.0.2.Final…) of the upstream Snowdrop BOM release to update"'